packages/cli/src/commands/start.ts

```diff
@@ -167,7 +169,7 @@ export default class Start extends Command {
         : '',
       logTransaction: environmentInfo.logTransaction,
       fileTransportsOptions: [
-        { filename: join(Config.logsPath, `${environmentInfo.name}-out.log`) }
+        { filename: environmentInfo.disableLogToFile ? '/dev/null' : join(Config.logsPath, `${environmentInfo.name}-out.log`) }
       ]
```

Comment:
using `/dev/null` on Windows will actually create a `null` file and still do the logging. 
Keeping the array empty instead should work on all platforms.

Reply:
Unfortunately that is correct, fixed by keeping the array empty when we want to disable logging to a file